### PR TITLE
Prep for 4.0.0 release

### PR DESCRIPTION
**Pull request by saifshaikh48**
This PR updates the WMCO version to v4.0.0 and updates the olm.SkipRange to '>=3.0.0 <4.0.0' for the 4.9 operator release.

### CODE REVIEW

File: config/rbac/role.yaml [hunk 1]
```diff
@@ -129,6 +130,7 @@ rules:
   resources:
   - signers
   verbs:
+  - Approve
   - approve
```
Comment (sebsoto) [inline config/rbac/role.yaml:134]: Where is this coming from? This isnt just `release prep` when this is a functionality change.

**Review by sebsoto — CHANGES_REQUESTED** [review]

File: config/rbac/role.yaml (hunk 1 above)
Comment (mansikulkarni96) [reply config/rbac/role.yaml:134]: I don't think that's required.

Comment (saifshaikh48) [reply config/rbac/role.yaml:134]: Seems to have been generated when I ran `make bundle`

Comment (saifshaikh48) [reply config/rbac/role.yaml]: @sebsoto @mansikulkarni96 I've manually removed the role changes.

**Comment by mansikulkarni96** [comment]
Thanks for adding this @saifshaikh48 
/lgtm

**Comment by saifshaikh48** [comment]
/test aws-e2e-upgrade
machine provisioning failure - retest

**Review by aravindhp — APPROVED** [review]
/approve

**Pull request merged**